### ISSUE TITLE
Newline bug has returned? TW 2.6.1

### ISSUE THREAD

**Issue by hymie0** [thehunmonkgroup/onenote#3]
I think the newline bug may have returned to Taskwarrior 2.6.1 .  All of my onenote notes are reverting to a single line with `\n` characters where the newlines used to be.  As a test, I set

```
TASK_VERSION_NO_NEWLINE="2.7"
```

and onenote works as expected, keeping the newlines in place.

**Comment by thehunmonkgroup** [comment]
https://github.com/GothenburgBitFactory/taskwarrior/issues/2645

I'm not running 2.6.x because of this active bug. Last properly working version of TW that I've tested is 2.5.3, waiting on a maintainer fix.

**Comment by duckunix** [comment]
I have a temporary work around until things get settled on the TW side:

```
diff onenote ~/bin/onenote
9c9
< newline="\n"
---
> newline="\\n"
137c137,138
<       echo "${notes}" | sed "s/${newline}/\\$linefeed/g" | ONENOTE_TASK=${task_number} ${pipe_editor} | ${0} ${task_number} ${consume_arg}
---
>       #echo "${notes}" | sed "s/${newline}/\\$linefeed/g" | ONENOTE_TASK=${task_number} ${pipe_editor} | ${0} ${task_number} ${consume_arg}
>       echo "${notes}" | sed 's/\\n/\n/g' | ONENOTE_TASK=${task_number} ${pipe_editor} | ${0} ${task_number} ${consume_arg}
```

**Comment by thehunmonkgroup** [comment]
@duckunix @hymie0 Pretty sure the 1.1.0 release fixes this robustly for 2.5.x and 2.6.x -- let me know if there are any other issues.
